lib/Request.js

```diff
@@ -67,11 +67,11 @@ class Request extends CallableInstance {
       args.push(undefined)
     }
 
-    if (!bubble.active) args[2] = true
+    if (!this._remit._namespace.active) args[2] = true
```

Comment:
This can potentially throw if someone passes a non-object where we expect a context. Could be a non-issue just mentioning it in-case :)

Reply:
Aye I'm sort of okay with it breaking, though it _would_ be easy enough to include a check that confirms it has everything we need - `active`, `get()`, `set()`, `runAndReturn()`, and `bind()`.

Sensibly, though, we'd then need to cover the same bases with the passed-in `tracer`, I guess? I feel like maybe it's okay just letting it die if somebody buggers up the config. :D